routes/toSwipe.js

```diff
@@ -27,13 +26,27 @@ router.get('/', ensureAuthenticated, (req, res) => {
 
       // fetch all movies in lists by active filter as well as default filters
       MovieList.find(
-        { $or: [{ filterName: { $in: filters } }, { filterType: 'default' }] },
+        { filterName: { $in: genreFilters } },
         async (error, doc) => {
           try {
             if (err) throw err;
             if (!doc) return [];
             if (doc) {
-              const moviesList = doc[0].films;
+              if (doc.length === 0) {
+                return [];
+              }
+              const tempMoviesList = doc[0].films;
+              const years = timeFilters.map(
+                (x) => parseInt(x.slice(0, -1)) / 10
+              );
```

Comment:
you can replace this with 
``` js
const years = timeFilters.map((x) => Number(x))
```
it's much simpler to understand and does the same thing